app/responders/ropensci/submit_review_responder.rb

```diff
@@ -7,13 +7,18 @@ class SubmitReviewResponder < Responder
 
     def define_listening
       @event_action = "issue_comment.created"
-      @event_regex = /\A@#{bot_name} submit review (\S+) time ([\d.,]+)+\s?(h|hour|hours)?\.?\s*\z/i
+      @event_regex = /\A@#{bot_name} submit review (\S+) time ([\d.,:]+)+\s?(h|hour|hours)?\.?\s*\z/i
     end
 
     def process_message(message)
       review_url = @match_data[1]
       review_time = @match_data[2].to_s.gsub(",", ".")
 
+      if review_time.include?(":")
+        respond("Error: Invalid time format")
```

Comment:
Great @xuanxu, that's exactly what i was imaging! I think that will be very useful - thank you